README.rst

```diff
@@ -57,10 +57,10 @@ There are three types of build that can be created from this codebase.
 2. Multi-paged documentation build of the normative specification.
 3. Single page documentation build of the normative specification.
 
-The different builds can be controlled through the use of the environment variable:
+The different builds can be controlled through the use of the environment variable::
```

Comment:
Why two columns?

Reply:
colons? to make the next section a command section.  Standard rst.

Reply:
Colons, yes (doh!). And, yes, indeed about the command section (I personally put it on a. new line, so got confused). Still, why not simply:

The different builds can be controlled through the use of the environment variable ``CELLML_SPEC_BUILD``.